.github/workflows/schedule.yml

```diff
@@ -1,15 +1,18 @@
 name: Update gist with WakaTime stats
 on:
   schedule:
+    # The gist is set to update once a day, which is the rate which the Wakatime API updates for this endpoint.
+    # You won't gain anything by increasing the rate of the scheduled task.
+    # Reference: https://github.com/matchai/waka-box/issues/49
     - cron: "0 0 * * *"
 jobs:
   update-gist:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@master
       - name: Update gist
-        uses: matchai/waka-box@master
+        uses: rlam3/waka-box@master
         env:
           GH_TOKEN: ${{ secrets.GH_TOKEN }}
-          GIST_ID: 968220c97e8da1d047a9a480fa432e54
+          GIST_ID: 9e951e17fd4f1c62a32fdc8d7eff3bed
```

Comment:
DO NOT take your personal stuff into code review.

Reply:
Rightly said.